llvm/test/CodeGen/AMDGPU/machine-scheduler-sink-trivial-remats.mir

```diff
@@ -6047,9 +7328,9 @@ body:             |
 
   bb.4:
 
+    S_NOP 0, implicit %52, implicit %62
```

Comment:
In a few cases here and below I moved this instruction to the top of the region to allow rematerializations to increase occupancy. Otherwise the instruction defining %60 is rematerialized at the beginning of the region which doesn't decrease maximum RP in the region, even though my optimistic heuristics estimate that it should have been enough. Ideally, I think the scheduler should be able to reorder instructions in such cases, but it does not. Alternatively, we could make the stage perform more rematerializations when it realizes pre-rescheduling that it did not achieve the expected goal.